activerecord/test/cases/schema_loading_test.rb

```diff
@@ -0,0 +1,53 @@
+require 'thread'
```

Comment:
@kbrock How are we relying on this here? See https://github.com/rails/rails/pull/24790